client/src/Components/Box/Box.tsx

```diff
@@ -24,9 +32,11 @@ export const Box: FC<BoxComponentProps> = ({ title, author, rate, review, image
         <BookParagraph>Średnia ocena: {rate}</BookParagraph>
         <BoxBookReview>
           {review}
-          <BoxReadMore onClick={readMoreHandler} data-testid="BoxReadMoreId">czytaj więcej</BoxReadMore>
+          <BoxReadMore onClick={readMoreHandler} data-testid="BoxReadMoreId">
+            czytaj więcej
```

Comment:
Może jako props? Żeby było to bardziej uniwersalne

Reply:
@WarmuzWojtek